tests/integration/src/test/resources/org/mule/test/integration/xml/xslt-functional-test-flow.xml

```diff
@@ -17,7 +17,7 @@
     <!--// START SNIPPET: transformer-config -->
     <mulexml:xslt-transformer name="xslt">
         <mulexml:xslt-text>
-            <xsl:stylesheet version="2.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+            <xsl:stylesheet version="2.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns="">
```

Comment:
Because xsl stylesheet is embedded as it is on this sample configuration the xmlns used for the output transformation would be the one set in the xmlns attribute or the default one that is inherited by the xml where the stylesheet is defined. Here it would be mule/core due to it is set as default xmlns because it is embedded in mule configuration. It is actually better to have this in a separate xml.

This is how currently works but when you look at the junit that is doing the assertion for this transformation it is not validating the xmlns attribute in output because of the conflicting libraries issue with XmlUnit, therefore with Saxon upgraded version here the test started to fail when comparing expected and result transformation.